aptos-move/framework/cached-packages/build.rs

```diff
@@ -12,6 +12,14 @@ fn main() {
         // Get the previous directory
         let mut prev_dir = current_dir;
         prev_dir.pop();
+        println!(
+            "cargo:rerun-if-changed={}",
+            prev_dir.join("aptos-token-objects").join("Move.toml").display()
+        );
+        println!(
+            "cargo:rerun-if-changed={}",
+            prev_dir.join("aptos-token-objects").join("sources").display()
+        );
```

Comment:
We should probably make this shorter to update in the future

Reply:
should be a trivial macro